src/components/BankIconButton.vue

```diff
@@ -32,9 +36,17 @@ export default defineComponent({
     }
 
     svg {
-        height: 9rem;
+        height: 8.25rem;
         width: auto;
-        margin-top: -.5rem;
+    }
+
+    .triangle {
+        position: absolute;
+        right: 4rem;
+        top: 8rem;
+        width: 1.25rem;
+        height: 1rem;
+        opacity: 0.25;
```

Comment:
Would be nice to avoid the code duplication.
Maybe by moving the style `button .triangle-down-icon` to `TriangleDownIcon`?